src/commands/pinning/pinByHash.js

```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
 import { baseUrl } from './../../constants';
-import { validateApiKeys, validateHostNodes, validateMetadata, validatePinPolicyStructure } from '../../util/validators';
+import { validateApiKeys, validateMetadata } from '../../util/validators';
```

Comment:
Just removed the unused imports, the husky pre-commit hook should prevent this in the future